obspy/io/nordic/core.py

```diff
@@ -0,0 +1,1338 @@
+# -*- coding: utf-8 -*-
+"""
+NonLinLoc file format support for ObsPy
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+We have not implemented any handling of focal mechanism solutions between
+the two formats.
+
+.. note:: Pick time-residuals are handled in event.origins[0].arrivals, with \
+    the arrival.pick_id linking the arrival (which contain calculated \
+    information) with the pick.resource_id (where the pick contains only \
+    physical measured information).
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA @UnusedWildImport
+from six import string_types
+from obspy import UTCDateTime
+import numpy as np
+from obspy.core.event import Event, Origin, Magnitude, Comment, Catalog
+from obspy.core.event import EventDescription, CreationInfo
+from obspy.core.event import Pick, WaveformStreamID, Arrival, Amplitude
+import warnings
```

Comment:
Please change import order (and newlines) to

```python
from __future__ import (absolute_import, division, print_function,
                        unicode_literals)
from future.builtins import *  # NOQA @UnusedWildImport

import warnings

import numpy as np

from obspy import UTCDateTime
from obspy.core.event import Event, Origin, Magnitude, Comment, Catalog
from obspy.core.event import EventDescription, CreationInfo
from obspy.core.event import Pick, WaveformStreamID, Arrival, Amplitude
```

So its future -> builtins -> scipy stack -> obspy